src/components/App/App.js

```diff
@@ -2,42 +2,58 @@ import React, {Component} from 'react';
 import './App.css';
 import Form from '../Form/Form';
 import Movies from '../Movies/Movies';
-import movieData from '../../data/movieData';
 import SingleMovie from '../SingleMovie/SingleMovie';
+import getData from '../../apiCalls/api';
 
 class App extends Component {
  constructor() {
   super();
   this.state = {
     allMovies: [],
     singleMovieId: 0,
-    singleMovie: {},
+    singleMovie: "",
+    error: "",
+    isLoading: true,
   }
  }
 
  componentDidMount = () => {
-  this.setState({allMovies: movieData.movies})
+  
+  getData("movies")
```

Comment:
getData isn't the most descriptive function name, but the argument does help us know what kind of data we're getting. Just something to consider.